proto/otaclient_iot_logging_server_v1.proto

```diff
@@ -25,27 +25,21 @@ enum LogType {
   LOG = 0;
   METRICS = 1;
 }
-enum LogLevel {
-  UNSPECIFIC = 0;
-  TRACE = 1;
-  DEBUG = 2;
-  INFO = 3;
-  WARN = 4;
-  ERROR = 5;
-  FATAL = 6;
-}
+
 enum ErrorCode {
-  OK = 0;
-  FAILED = 1;
+  NO_FAILURE = 0;         // Success
+  SERVER_QUEUE_FULL = 1;  // Error: Server queue is full
+  NOT_ALLOWED_ECU_ID = 2; // Error: Specified ECU ID is not allowed
+  NO_MESSAGE = 3;         // Error: No message in the request
 }
+
 message PutLogRequest {
-  LogType type = 1;
-  uint64 timestamp = 2;
-  LogLevel level = 3;
-  string data = 4;
+  string ecu_id = 1;    // target ECU ID
+  LogType log_type = 2; // log type
+  string message = 3;   // log message
```

Comment:
At first, try to include `level` and `timestamp` fields.
But reconsider that it will be better to handle them in `ota-client` side(= otaclient-iot-logging-server doesn't care about log contents at all).
So removed these fields.

Reply:
I think otaclient-iot-logger still cares about level and timestamp.

Considering the following two case:
1. In the very worst scenario, there might be a buggy otaclient on an ECU which produces a lot of logs, flooding the otaclient-iot-logger.
2. the network is disconnected.

For each ECU, we have Log queue for that ECU, and apply the strategy per log queue to ensure fairness.

We want to send the log as much as possible, but when the otaclient-iot-logger log queues are full(due to network is disconnected or being flooded), for each ECU's log queue, we might want to adopt the following strategies:
1. high level messages like Error and above will never be dropped.
2. dropping DEBUG level unconditionally.
3. dropping INFO level with FIFO order using the timestamp in the message.

Reply:
Thank you. OK, let's add `level` and `timestamp`. Fixed.

Reply:
The strategy above will be introduced in the future PR, currently otaclient-iot-logger only has a single queue for all logs.